packages/renderer/src/lib/image/ImportContainersImages.svelte

```diff
@@ -0,0 +1,166 @@
+<script lang="ts">
+import { faMinusCircle, faPlay, faPlusCircle } from '@fortawesome/free-solid-svg-icons';
+import { Input } from '@podman-desktop/ui-svelte';
+import { get, onMount } from 'svelte';
+import { router } from 'tinro';
+
+import { providerInfos } from '/@/stores/providers';
+
+import type { ProviderContainerConnectionInfo, ProviderInfo } from '../../../../main/src/plugin/api/provider-info';
+import Button from '../ui/Button.svelte';
+import ErrorMessage from '../ui/ErrorMessage.svelte';
+import FormPage from '../ui/FormPage.svelte';
+
+let containersToImport: { imagePath: string; nameWhenImporting: string }[] = [];
+let importError: string = '';
+
+let providers: ProviderInfo[] = [];
+let providerConnections: ProviderContainerConnectionInfo[] = [];
+let selectedProvider: ProviderContainerConnectionInfo | undefined = undefined;
+let inProgress = false;
+
+$: importDisabled = !selectedProvider || containersToImport.length === 0;
+let separator = '/';
+
+onMount(async () => {
+  providers = get(providerInfos);
+  providerConnections = providers
+    .map(provider => provider.containerConnections)
+    .flat()
+    .filter(providerContainerConnection => providerContainerConnection.status === 'started');
+
+  const selectedProviderConnection = providerConnections.length > 0 ? providerConnections[0] : undefined;
+  selectedProvider = !selectedProvider && selectedProviderConnection ? selectedProviderConnection : selectedProvider;
+
+  const platform = await window.getOsPlatform();
+  if (platform === 'win32') {
+    separator = '\\';
+  }
+});
```

Comment:
the `openDialog` still returns a string. I just updated the saveDialog.
The separator is just needed to calculate an initial importing name, which could also be not completely correct. It is just to generate something to start with for the user but they can update it.
E.g. I just take the content starting from the last slash to the last dot. So let's say the path is path/file.tar.gz, the resulting importing image name would be `file.tar` while it would better be `file`